src/spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../person'
+
+describe Person do
+  context 'When passing a name' do
+    person = Person.new(id: nil, age: 22, name: 'leo', parent_permission: true)
+    it 'should return first letter capital' do
+      expect(person.validate_name).to eq 'Leo'
+    end
+
+    it 'should return true if he can you service or false' do
+      expect(person.can_use_services?).to eq true
+    end
```

Comment:
- [ ] great job on writing the tests 👍 however try to write tests for positive and negative case scenarios instead of only positive because we need to test how the app reacts when it is false.